src/client/components/ContactForm/ContactForm.styles.css

```diff
@@ -0,0 +1,47 @@
+form {
+  display: grid;
+  padding: 1em;
+  background: #f9f9f9;
+  border: 1px solid #c1c1c1;
+  margin: 2rem auto 0 auto;
+  max-width: 400px;
+  padding: 1em;
+}
+
+label {
+  padding: 0.5em 0.5em 0.5em 0;
+}
+
+@media (min-width: 400px) {
+  form {
+    grid-template-columns: 1fr 8fr;
+    grid-gap: 6px;
+    font-style: normal;
+    font-family: 'Inter', sans-serif;
+    font-size: 14px;
+  }
+
+  .title {
+    text-align: left;
+    grid-column: 2 / span 1;
+    grid-row: 1;
+    font-weight: bold;
+  }
+
+  .fullName {
+    text-align: left;
```

Comment:
If fullName and email have the same properties, you could select both at the same time or have them in a container.